pkg/resourcecollector/virtualmachine.go

```diff
@@ -25,7 +26,8 @@ const (
 	//VMUnFreezeCmd is the template for VM unfreeze command
 	VMUnFreezeCmd = "/usr/bin/virt-freezer --unfreeze --name %s --namespace %s"
 	// VMContainerName is the name of the container to use for freeze/thaw
-	VMContainerName = "compute"
+	VMContainerName             = "compute"
+	VMPodSelectorCreatedByLabel = "kubevirt.io/created-by"
```

Comment:
Would this label be unique for VM created across different ns?

Reply:
yes